test/webdriverio/test/flyout_test.ts

```diff
@@ -0,0 +1,290 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as chai from 'chai';
+import * as Blockly from 'blockly';
+import {
+  testSetup,
+  testFileLocations,
+  PAUSE_TIME,
+  tabNavigateForward,
+  getFocusedBlockType,
+  keyDown,
+  tabNavigateBackward,
+  tabNavigateToWorkspace,
+  keyRight,
+  getCurrentFocusNodeId,
+} from './test_setup.js';
+
+suite('Toolbox and flyout test', function () {
+  // Clear the workspace and load start blocks
+  setup(async function () {
+    this.browser = await testSetup(testFileLocations.BASE);
+    await this.browser.pause(PAUSE_TIME);
+  });
+
+  test('Tab navigating to toolbox should open flyout', async function () {
+    // Two tabs should navigate to the toolbox (initial element is skipped).
+    await tabNavigateForward(this.browser);
+
```

Comment:
Delete seemingly spurious blank line?  (Here and on several more test cases below).

Reply:
These actually aren't spurious. I usually format my tests always into two or three blocks for distinct arrange-act-assert sections (to make it very clear to readers which part of the test is setup, the actual operation being verified, and the code for doing that verifying).